src/common/src/monitor/process_linux.rs

```diff
@@ -1,209 +0,0 @@
-// Copyright 2023 RisingWave Labs
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-use prometheus::core::{Collector, Desc};
```

Comment:
How will we monitor the process after this PR?

Reply:
https://github.com/tikv/rust-prometheus/blob/master/src/registry.rs#L280-L305

The default registry already has process monitor for Linux.